DiffPlex/DiffBuilder/Model/DiffPaneModel.cs

```diff
@@ -1,10 +1,13 @@
-﻿using System.Collections.Generic;
+﻿using System;
+using System.Collections.Generic;
 using System.Linq;
 
 namespace DiffPlex.DiffBuilder.Model
 {
     public class DiffPaneModel
     {
+        public List<DiffPiece> Chunks => Lines;
```

Comment:
Why make an alias like this?

Reply:
Because when Line chunker isn't used as first the current name "Lines"  doesn't represent its contents.

But its just a small alias np for me to delete it.

Reply:
Sounds fine, can you put summary comments on both maybe to clear up any confusion?